src/content/contribute/writing-a-loader.md

```diff
@@ -218,59 +226,101 @@ __src/loader.js__
 ``` js
 import { getOptions } from 'loader-utils';
 
-export default function(source) {
+export default source => {
   const options = getOptions(this);
 
-  source.replace(/\[name\]/g, options.name);
+  source = source.replace(/\[name\]/g, options.name);
 
   return `export default ${ JSON.stringify(source) }`;
 };
 ```
 
-With that in place, we can start setting up our tests. We'll start by adding a configuration that utilizes this loader:
+We'll use this loader to process the following file:
+
+__test/example.txt__
+
+``` text
+Hey [name]!
+```
+
+Pay close attention to this next step as we'll be using the [Node.js API]() and [`memory-fs`]() to execute webpack. This lets us avoid emitting `output` to disk and will give us access to the `stats` data which we can use to grab our transformed module:
 
-__test/webpack.config.js__
+__test/compiler.js__
 
 ``` js
-const path = require('path');
-
-module.exports = {
-  entry: './example.txt',
-  output: {
-    path: path.resolve(__dirname),
-    filename: './built.js'
-  },
-  module: {
-    rules: [
-      {
+import path from 'path';
+import webpack from 'webpack';
+import memoryfs from 'memory-fs';
+
+export default () => {
```

Comment:
`export default (fixture, options = {}) => {`

Reply:
Ok that's fair, I'll change it and maybe pass the `./example.txt` string in instead of hardcoding it.